### PR TITLE
Multisig setup

### PR DESCRIPTION
**Pull request by j-kon**
**What does this PR do?**

A summary of what this PR is about

**List of task completed**
-   [x] Created intro screen
-   [x] Created a set-up screen
-   [x] Modify UI

On this branch I created the a set-up screen for multisig and also the adjust UI

**How can this be tested?**

- Highlight how this PR can be tested

**Additional info**
### 📸 ScreenShots

Add more info if any. eg fixes #.

### CODE REVIEW

File: Frontend/lib/screens/home/key_creation.dart [hunk 1]
```diff
@@ -0,0 +1,102 @@
+import 'package:flutter/material.dart';
```
Comment (toshmanuel) [inline Frontend/lib/screens/home/key_creation.dart:1]: why do we have this screen, I don't think we need to set up any cosigner.

**Review by toshmanuel — CHANGES_REQUESTED** [review]

**Review by toshmanuel — APPROVED** [review]

File: Frontend/lib/screens/auth/forget_password.dart [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                text: Text('Send Link',
+                    style: AppTextStyle.textSize21
+                        .copyWith(color: AppColors.whiteColor)),
+                height: 60,
+                width: 335,
+              ),
+              Height15(),
+              Center(
+                child: Text(
+                  'A link will will be send to your email for password reset',
```
Comment (Muneerah1612) [inline Frontend/lib/screens/auth/forget_password.dart]: A link will be sent,not send

File: Frontend/lib/utils/constant.dart [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+const String forgotPasswordText = 'Forgot Password?';
+const String dontHaveAnAccountText = 'Don\'t have an account?';
+const String alreadyHaveAnAccountText = 'Already have an account?';
+const String howItWorkText = 'How it works';
+const String eachCosignerText = 'Each cosigner needs to create their own key on their device.';
+const String allCosignerText = 'All cosigners exchange keystores.';
+const String withAll3Text = 'With all 3 keystores, each cosigner can set up the wallet on their device and view the balance and transactions.';
+const String createSharedWallet = 'Create shared wallet';
+const String recoverWalletYou = 'Recover a wallet you previously used.';
+const String summitYourText = 'Summit Your Key';
```
Comment (Muneerah1612) [inline Frontend/lib/utils/constant.dart]: 'Submit', always check your spelling

**Review by Muneerah1612 — APPROVED** [review]

File: Frontend/lib/screens/auth/forget_password.dart (hunk 2 above)
Comment (j-kon) [reply Frontend/lib/screens/auth/forget_password.dart]: > A link will be sent,not send

Thanks resolve

File: Frontend/lib/utils/constant.dart (hunk 3 above)
Comment (j-kon) [reply Frontend/lib/utils/constant.dart]: > 'Submit', always check your spelling

Thanks resolve

**Comment by stickies-v** [comment]
Thanks for adding screenshots to the PR! Going forward I'd avoid having bad commits in your PRs altogether. All the "added to gitignore" commits should have just been dropped or squashed before merging - plus there are still some .pyc files checked in that shouldn't be there.

**Comment by j-kon** [comment]
> Thanks for adding screenshots to the PR! Going forward I'd avoid having bad commits in your PRs altogether. All the "added to gitignore" commits should have just been dropped or squashed before merging - plus there are still some .pyc files checked in that shouldn't be there.

Well noted, thanks

**Pull request merged**